fastlane/Fastfile

```diff
@@ -36,6 +36,34 @@ def get_required_env(key)
   ENV[key]
 end
 
+# Helper function to DRY the code to build the app
+def build_simplenote(codesign:, archive_path: nil)
+  if codesign
+    skip_archive = false
+    xcargs = []
+  else
+    # Skip archiving when not codesigning
+    skip_archive = true
+    xcargs = @SKIP_CODE_SIGNING_XCARGS
+  end
+
+  build_mac_app(
+    workspace: @WORKSPACE,
+    scheme: @APP_STORE_SCHEME,
+    configuration: 'Release',
+    clean: true,
+    output_directory: @BUILD_FOLDER,
+    archive_path: archive_path,
+    skip_archive: skip_archive,
+    export_method: 'mac-application',
+    export_team_id: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
```

Comment:
Other projects read this value from an encrypted dotfile. I'm not sure why, since the value is tracked in clear as part of the `Appfile`.

I don't have a problem with using the encrypted dotfile. I didn't set it up here because it's extra work... 😅 If we want to do that, though, should we then remove the value from the `Appfile`? I don't think it's a security risk. Having the team identifier is useless without an account (_last famous words?_)

Reply:
Yeah I don't see the point of having the team ID be encrypted and pulled from secrets. I like your solution here better.

Reply:
I agree with this – the team ID is not a secret, no need to treat it as one. I'd even go one step further and say "let's get rid of the `Appfile` – IMHO having a ton of files that just have configuration is less good than having one file that has all the lanes + some configuration at the top that's shared between all of them.

Doesn't have to be part of this PR though.